src/Get.tsx

```diff
@@ -214,7 +232,7 @@ function Get<TData = any, TError = any>(props: GetProps<TData, TError>) {
   return (
     <RestfulReactConsumer>
       {contextProps => (
-        <RestfulReactProvider {...contextProps} base={`${contextProps.base}${props.path}`}>
+        <RestfulReactProvider {...contextProps} base={`${contextProps.base}/${props.path}`}>
```

Comment:
Should concatenation happen here regardless of whether the `path` prop has a leading slash or not? I wonder how relativeness is reconciled further down the component tree.

Reply:
Could you expand more @peterszerzo ? If we have no leading slash and concat it would look like `https://example.complop`. Maybe you are saying if I want to do a relative path but started out with a parent like `/plop` then the concatenation would be `https://example.com//plop`.

I may be missing a normalization of the path here.

Reply:
What seems more intuitive to me is that if the `path` prop starts with a leading slash, the `base` prop of the context provider stays unmodified. This would allow child components to work with their grandparents' `base` prop, essentially dropping whatever the parent has set (as opposed to picking up on it in case the `path` didn't start with a slash).

Reply:
I am still not seeing the absolute path escape through a method like this. I must fully be overlooking some context here.

if the base url is `https://example.com`

```jsx
<Get path="/absolute"> // fetch 'https://example.com/absolute'
  <Get path="relative" > // fetch 'https://example.com/relative' (because the absolute didn't get appended)
    <Get path="again" > // fetch 'https://example.com/relative/again'
      // how do we know what the base is if the base got modified?
      // What absolute path do I fetch here? At this point base
      // would be https://exapmle.com/relative/again
       <Get path="/more-absolute" />
   </Get>
 </Get>
</Get>
```

Reply:
I see! I misunderstood. I thought you were saying concatenating was the solution without the `originalBase`. My mistake.

The semantics around fetching here are complex. My opinion would be that nested routes are always relative regardless of leading slash. If you wanted to make an internal route fetch escape to the base URL then a prop of absolute might make it more clear?

```jsx
<Get path="/absolute"> // fetch 'https://example.com/absolute'
  <Get path="relative" > // fetch 'https://example.com/absolute/relative'
    <Get path="again" > // fetch 'https://example.com/absolute/relative/again'
       // This would hit the originalBase
       <Get absolute path="more-absolute" />
   </Get>
 </Get>
</Get>
```

But I do not have enough context on the usage patterns of this lib I am not sure what the implicit interpretation of the routes with leading slashes is in the "wild" 😆

Reply:
Awesome, I think this is a lot better - the presence or absence of a slash is too little visual noise for difference in behavior so significant. I would actually be in favor of recommending slash-less `path` throughout the usage of the library (whether through warnings or just phrasing all examples that way).